### PR TITLE
New instrument cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[PYV Instrument Cal CSVs [DT-184]](https://trello.com/c/Eo546k32/184-pyv-instrument-cal-csvs-dt-184)

File: calibration/DOSTAD/CGINS-DOSTAD-00224__20230505.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+224,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_224_Multipoint_Calibration_2023-05-05.pdf
+224,CC_csv,"[2.82689E-03, 1.20089E-04, 2.27522E-06, 1.85755E+02, -2.27536E-01, -4.43958E+01, 3.63280E+00]",
```
Comment (sbeaulieu) [inline calibration/DOSTAD/CGINS-DOSTAD-00224__20230505.csv:3]: Looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00226__20230505.csv [hunk 2]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+226,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_226_Multipoint_Calibration_2023-05-05.pdf
+226,CC_csv,"[2.62319E-03, 1.11562E-04, 2.20510E-06, 1.97522E+02, -2.54835E-01, -4.79570E+01, 3.91605E+00]",
```
Comment (sbeaulieu) [inline calibration/DOSTAD/CGINS-DOSTAD-00226__20230505.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00379__20230505.csv [hunk 3]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+379,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_379_Multipoint_Calibration_2023-05-05.pdf
+379,CC_csv,"[2.49522E-03, 1.08411E-04, 1.79483E-06, 2.04227E+02, -2.40200E-01, -4.66475E+01, 4.03495E+00]",
```
Comment (sbeaulieu) [inline calibration/DOSTAD/CGINS-DOSTAD-00379__20230505.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00385__20230505.csv [hunk 4]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+385,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_385_Multipoint_Calibration_2023-05-05.pdf"
+385,CC_csv,"[2.56958E-03, 1.09730E-04, 2.06051E-06, 2.01644E+02, -2.95287E-01, -5.05581E+01, 4.03246E+00]",
```
Comment (sbeaulieu) [inline calibration/DOSTAD/CGINS-DOSTAD-00385__20230505.csv:3]: looks good

File: calibration/ADCPSL/CGINS-ADCPSL-21446__20230609.csv [hunk 5]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+21446,CC_scale_factor1,0.45,constant; source_file ADCPS-L_WHLS75-1500-I_SN_21446_Configuration_Summary_2023-06-09.pdf
+21446,CC_scale_factor2,0.45,constant
+21446,CC_scale_factor3,0.45,constant
+21446,CC_scale_factor4,0.45,constant
+21446,CC_bin_size,0,in cm; return from vendor - not allocated yet
+21446,CC_dist_first_bin,0,in cm; return from vendor - not allocated yet
+21446,CC_orientation,1,1 is upward looking; 0 is downward looking
```
Comment (sbeaulieu) [inline calibration/ADCPSL/CGINS-ADCPSL-21446__20230609.csv:8]: confirmed with Rebecca over Slack, looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00386__20230505.csv [hunk 6]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+386,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_386_Multipoint_Calibration_2023-05-05.pdf
+386,CC_csv,"[2.56178E-03, 1.09107E-04, 2.10725E-06, 5.75669E+01, -7.21496E-02, -1.42368E+01, 1.14979E+00]",
```
Comment (sbeaulieu) [inline calibration/DOSTAD/CGINS-DOSTAD-00386__20230505.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00393__20230505.csv [hunk 7]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+393,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_393_Multipoint_Calibration_2023-05-05.pdf
+393,CC_csv,"[2.60770E-03, 1.11383E-04, 2.11751E-06, 2.14793E+02, -3.11778E-01, -5.42762E+01, 4.27312E+00]",
```
Comment (sbeaulieu) [inline calibration/DOSTAD/CGINS-DOSTAD-00393__20230505.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00413__20230505.csv [hunk 8]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+413,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_413_Multipoint_Calibration_2023-05-05.pdf
+413,CC_csv,"[2.54000E-03, 1.08323E-04, 2.04878E-06, 1.79893E+02, -2.32320E-01, -4.25315E+01, 3.58009E+00]",
```
Comment (sbeaulieu) [inline calibration/DOSTAD/CGINS-DOSTAD-00413__20230505.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00415__20230505.csv [hunk 9]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+415,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_415_Multipoint_Calibration_2023-05-05.pdf
+415,CC_csv,"[2.60445E-03, 1.10694E-04, 2.09807E-06, 1.96260E+02, -2.68756E-01, -4.67264E+01, 3.90159E+00]",
```
Comment (sbeaulieu) [inline calibration/DOSTAD/CGINS-DOSTAD-00415__20230505.csv:3]: looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01065__20230427.csv [hunk 10]
```diff
@@ -0,0 +1,8 @@
...
....92, 274.74, 275.55, 276.37, 277.18, 278, 278.81, 279.63, 280.44, 281.26, 282.07, 282.88, 283.7, 284.51, 285.33, 286.14, 286.96, 287.77, 288.59, 289.4, 290.22, 291.03, 291.85, 292.66, 293.48, 294.29, 295.11, 295.92, 296.74, 297.55, 298.37, 299.18, 299.99, 300.81, 301.62, 302.44, 303.25, 304.07, 304.88, 305.69, 306.51, 307.32, 308.14, 308.95, 309.76, 310.58, 311.39, 312.21, 313.02, 313.83, 314.65, 315.46, 316.27, 317.09, 317.9, 318.71, 319.53, 320.34, 321.15, 321.96, 322.78, 323.59, 324.4, 325.21, 326.03, 326.84, 327.65, 328.46, 329.27, 330.09, 330.9, 331.71, 332.52, 333.33, 334.14, 334.95, 335.77, 336.58, 337.39, 338.2, 339.01, 339.82, 340.63, 341.44, 342.25, 343.06, 343.87, 344.68, 345.49, 346.3, 347.11, 347.91, 348.72, 349.53, 350.34, 351.15, 351.96, 352.76, 353.57, 354.38, 355.19, 355.99, 356.8, 357.61, 358.42, 359.22, 360.03, 360.84, 361.64, 362.45, 363.25, 364.06, 364.86, 365.67, 366.47, 367.28, 368.08, 368.89, 369.69, 370.5, 371.3, 372.1, 372.91, 373.71, 374.51, 375.32, 376.12, 376.92, 377.72, 378.52, 379.33, 380.13, 380.93, 381.73, 382.53, 383.33, 384.13, 384.93, 385.73, 386.53, 387.33, 388.13, 388.93, 389.73, 390.53, 391.32, 392.12, 392.92, 393.72, 394.51, 395.31, 396.11]",
```
Comment (sbeaulieu) [inline calibration/NUTNRB/CGINS-NUTNRB-01065__20230427.csv:8]: used EXCEL macro to check starting and ending values, looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01086__20230502.csv [hunk 11]
```diff
@@ -0,0 +1,8 @@
...
....56, 273.37, 274.17, 274.98, 275.78, 276.59, 277.39, 278.2, 279.01, 279.81, 280.62, 281.42, 282.23, 283.03, 283.84, 284.64, 285.45, 286.25, 287.06, 287.86, 288.67, 289.47, 290.28, 291.08, 291.89, 292.69, 293.5, 294.3, 295.11, 295.91, 296.72, 297.52, 298.33, 299.13, 299.94, 300.74, 301.55, 302.35, 303.15, 303.96, 304.76, 305.57, 306.37, 307.17, 307.98, 308.78, 309.58, 310.39, 311.19, 311.99, 312.8, 313.6, 314.4, 315.21, 316.01, 316.81, 317.61, 318.42, 319.22, 320.02, 320.82, 321.63, 322.43, 323.23, 324.03, 324.83, 325.63, 326.44, 327.24, 328.04, 328.84, 329.64, 330.44, 331.24, 332.04, 332.84, 333.64, 334.44, 335.24, 336.04, 336.84, 337.64, 338.44, 339.24, 340.04, 340.84, 341.63, 342.43, 343.23, 344.03, 344.83, 345.63, 346.42, 347.22, 348.02, 348.81, 349.61, 350.41, 351.21, 352, 352.8, 353.59, 354.39, 355.19, 355.98, 356.78, 357.57, 358.37, 359.16, 359.96, 360.75, 361.54, 362.34, 363.13, 363.92, 364.72, 365.51, 366.3, 367.1, 367.89, 368.68, 369.47, 370.26, 371.06, 371.85, 372.64, 373.43, 374.22, 375.01, 375.8, 376.59, 377.38, 378.17, 378.96, 379.75, 380.54, 381.32, 382.11, 382.9, 383.69, 384.47, 385.26, 386.05, 386.84, 387.62, 388.41, 389.19, 389.98, 390.76, 391.55, 392.33, 393.12]",
```
Comment (sbeaulieu) [inline calibration/NUTNRB/CGINS-NUTNRB-01086__20230502.csv:8]: used EXCEL macro to check starting and ending values, looks good

File: calibration/PHSENE/CGINS-PHSENE-P0139__20230504.csv [hunk 12]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+P0139,CC_ea434,17372.0, source_file PHSEN-E_SAMI-pH_SN_P0139_Validation_Files_2023-05-04.zip > SAMI_P0139_040523T_ValCert.pdf
+P0139,CC_ea578,94.1,
+P0139,CC_eb434,2284.0,
+P0139,CC_eb578,38676.0,
+P0139,CC_ind_off,0,
+P0139,CC_ind_slp,1,
+P0139,CC_psal,35,Constant
+P0139,CC_sami_bits,12,
```
Comment (sbeaulieu) [inline calibration/PHSENE/CGINS-PHSENE-P0139__20230504.csv:9]: confirmed Rev E via Rebecca Slack, looks good

File: calibration/PCO2WB/CGINS-PCO2WB-C0112__20230321.csv [hunk 13]
```diff
@@ -0,0 +1,11 @@
...
+C0112,CC_cala,0.0807,source_file PCO2W-B_SAMI-CO2_SN_C0112_Calibration_Files_2023-03-21.zip > C0112_210323_CalCert.pdf
+C0112,CC_calb,0.4642,
+C0112,CC_calc,-1.3498,
+C0112,CC_calt,6.8512,
+C0112,CC_ea434,19706,constant
+C0112,CC_ea620,34,constant
+C0112,CC_eb434,3073,constant
+C0112,CC_eb620,44327,constant
+C0112,CC_sami_bits,12,
+C0112,CC_cal_range,"[205,988]",
```
Comment (sbeaulieu) [inline calibration/PCO2WB/CGINS-PCO2WB-C0112__20230321.csv:11]: confirmed Rev E for sami_bits, looks good

File: calibration/PHSENE/CGINS-PHSENE-P0178__20230504.csv [hunk 14]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+P0178,CC_ea434,17372.0,source_file PHSEN-E_SAMI-pH_SN_P0178_Validation_Files_2023-05-04.zip > SAMI_P0178_040523T_ValCert.pdf
+P0178,CC_ea578,94.1,
+P0178,CC_eb434,2284.0,
+P0178,CC_eb578,38676.0,
+P0178,CC_ind_off,0,
+P0178,CC_ind_slp,1,
+P0178,CC_psal,35,Constant
+P0178,CC_sami_bits,14,
```
Comment (sbeaulieu) [inline calibration/PHSENE/CGINS-PHSENE-P0178__20230504.csv:9]: looks good Rev K

File: calibration/PHSENE/CGINS-PHSENE-P0140__20230609.csv [hunk 15]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+P0140,CC_ea434,17372.0,source_file PHSEN-E_SAMI-pH_SN_P0140_Validation_Files_2023-06-09.zip > SAMI_P0140_090623t_ValCert.pdf
+P0140,CC_ea578,94.1,
+P0140,CC_eb434,2284.0,
+P0140,CC_eb578,38676.0,
+P0140,CC_ind_off,0,
+P0140,CC_ind_slp,1,
+P0140,CC_psal,35,Constant
+P0140,CC_sami_bits,12,
```
Comment (sbeaulieu) [inline calibration/PHSENE/CGINS-PHSENE-P0140__20230609.csv:9]: looks good

File: calibration/PHSENF/CGINS-PHSENF-P0081__20230412.csv [hunk 16]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+P0081,CC_ea434,17372.0,source_file PHSEN-F_SAMI-pH_SN_P0081_Validation_Files_2023-04-12.zip > SAMI_P0081_120423t_ValCert.pdf
+P0081,CC_ea578,94.1,
+P0081,CC_eb434,2284.0,
+P0081,CC_eb578,38676.0,
+P0081,CC_ind_off,0,
+P0081,CC_ind_slp,1,
+P0081,CC_psal,35,Constant
+P0081,CC_sami_bits,12,
```
Comment (sbeaulieu) [inline calibration/PHSENF/CGINS-PHSENF-P0081__20230412.csv:9]: looks good

File: calibration/PHSENF/CGINS-PHSENF-P0103__20230526.csv [hunk 17]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+P0103,CC_ea434,17372.0, source_file PHSEN-F_SAMI-pH_SN_P0103_Validation_Files_2023-05-26.zip > SAMI_P0103_260523t_ValCert.pdf
+P0103,CC_ea578,94.1,
+P0103,CC_eb434,2284.0,
+P0103,CC_eb578,38676.0,
+P0103,CC_ind_off,0,
+P0103,CC_ind_slp,1,
+P0103,CC_psal,35,Constant
+P0103,CC_sami_bits,12,
```
Comment (sbeaulieu) [inline calibration/PHSENF/CGINS-PHSENF-P0103__20230526.csv:9]: looks good

File: calibration/PHSENF/CGINS-PHSENF-P0105__20230412.csv [hunk 18]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+P0105,CC_ea434,17372.0,source_file PHSEN-F_SAMI-pH_SN_P0105_Validation_Files_2023-04-12.zip > SAMI_P0105_120423t_ValCert.pdf
+P0105,CC_ea578,94.1,
+P0105,CC_eb434,2284.1,
+P0105,CC_eb578,38676.5,
+P0105,CC_ind_off,0,
+P0105,CC_ind_slp,1,
+P0105,CC_psal,35,Constant
+P0105,CC_sami_bits,12,
```
Comment (sbeaulieu) [inline calibration/PHSENF/CGINS-PHSENF-P0105__20230412.csv:9]: looks good

File: calibration/PHSENF/CGINS-PHSENF-P0141__20230412.csv [hunk 19]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+P0141,CC_ea434,17372.0,source_file PHSEN-F_SAMI-pH_SN_P0141_Validation_Files_2023-04-12.zip > SAMI_P0141_120423t_ValCert.pdf
+P0141,CC_ea578,94.1,
+P0141,CC_eb434,2284.0,
+P0141,CC_eb578,38676.0,
+P0141,CC_ind_off,0,
+P0141,CC_ind_slp,1,
+P0141,CC_psal,35,Constant
+P0141,CC_sami_bits,12,
```
Comment (sbeaulieu) [inline calibration/PHSENF/CGINS-PHSENF-P0141__20230412.csv:9]: looks good

File: calibration/PHSENF/CGINS-PHSENF-P0142__20230419.csv [hunk 20]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+P0142,CC_ea434,17372.0,source_file PHSEN-F_SAMI-pH_SN_P0142_Validation_Files_2023-04-19.zip > SAMI_P0142_190423t_ValCert.pdf
+P0142,CC_ea578,94.1,
+P0142,CC_eb434,2284.0,
+P0142,CC_eb578,38676.0,
+P0142,CC_ind_off,0,
+P0142,CC_ind_slp,1,
+P0142,CC_psal,35,Constant
+P0142,CC_sami_bits,12,
```
Comment (sbeaulieu) [inline calibration/PHSENF/CGINS-PHSENF-P0142__20230419.csv:9]: looks good

File: calibration/PHSENF/CGINS-PHSENF-P0143__20230526.csv [hunk 21]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+P0143,CC_ea434,17372.0, source_file PHSEN-F_SAMI-pH_SN_P0143_Validation_Files_2023-05-26.zip > SAMI_P0143_260523t_ValCert.pdf
+P0143,CC_ea578,94.1,
+P0143,CC_eb434,2284.0,
+P0143,CC_eb578,38676.0,
+P0143,CC_ind_off,0,
+P0143,CC_ind_slp,1,
+P0143,CC_psal,35,Constant
+P0143,CC_sami_bits,12,
```
Comment (sbeaulieu) [inline calibration/PHSENF/CGINS-PHSENF-P0143__20230526.csv:9]: looks good

File: calibration/PCO2WC/CGINS-PCO2WC-C0106__20230330.csv [hunk 22]
```diff
@@ -0,0 +1,11 @@
...
+C0106,CC_cala,0.0683,source_file PCO2W-C_SAMI-CO2_SN_C0106_Calibration_Files_2023-03-30.zip > C0106_300323_CalCert.pdf
+C0106,CC_calb,0.5058,
+C0106,CC_calc,-1.3234,
+C0106,CC_calt,6.8099,
+C0106,CC_ea434,19706,constant
+C0106,CC_ea620,34,constant
+C0106,CC_eb434,3073,constant
+C0106,CC_eb620,44327,constant
+C0106,CC_sami_bits,12,
+C0106,CC_cal_range,"[202,991]",
```
Comment (sbeaulieu) [inline calibration/PCO2WC/CGINS-PCO2WC-C0106__20230330.csv:11]: looks good

File: calibration/PCO2WC/CGINS-PCO2WC-C0136__20230309.csv [hunk 23]
```diff
@@ -0,0 +1,11 @@
...
+C0136,CC_cala,0.0590,source_file PCO2W-C_SAMI-CO2_SN_C0136_Calibration_Files_2023-03-09.zip > C0136_090323_CalCert.pdf
+C0136,CC_calb,0.5825,
+C0136,CC_calc,-1.4786,
+C0136,CC_calt,6.5126,
+C0136,CC_ea434,19706,constant
+C0136,CC_ea620,34,constant
+C0136,CC_eb434,3073,constant
+C0136,CC_eb620,44327,constant
+C0136,CC_sami_bits,12,
+C0136,CC_cal_range,"[205,987]",
```
Comment (sbeaulieu) [inline calibration/PCO2WC/CGINS-PCO2WC-C0136__20230309.csv:11]: looks good

File: calibration/PCO2WC/CGINS-PCO2WC-C0139__20230309.csv [hunk 24]
```diff
@@ -0,0 +1,11 @@
...
+C0139,CC_cala,0.0741,source_file PCO2W-C_SAMI-CO2_SN_C0139_Calibration_Files_2023-03-09.zip > C0139_090323_CalCert.pdf
+C0139,CC_calb,0.5234,
+C0139,CC_calc,-1.4403,
+C0139,CC_calt,6.5236,
+C0139,CC_ea434,19706,constant
+C0139,CC_ea620,34,constant
+C0139,CC_eb434,3073,constant
+C0139,CC_eb620,44327,constant
+C0139,CC_sami_bits,12,
+C0139,CC_cal_range,"[205,987]",
```
Comment (sbeaulieu) [inline calibration/PCO2WC/CGINS-PCO2WC-C0139__20230309.csv:11]: looks good

**Review by sbeaulieu — APPROVED** [review]

File: calibration/PHSENE/CGINS-PHSENE-P0139__20230504.csv (hunk 12 above)
Comment (cdobs) [inline calibration/PHSENE/CGINS-PHSENE-P0139__20230504.csv:9]: looks good

File: calibration/PHSENE/CGINS-PHSENE-P0140__20230609.csv (hunk 15 above)
Comment (cdobs) [inline calibration/PHSENE/CGINS-PHSENE-P0140__20230609.csv:9]: looks good

File: calibration/PHSENE/CGINS-PHSENE-P0178__20230504.csv (hunk 14 above)
Comment (cdobs) [inline calibration/PHSENE/CGINS-PHSENE-P0178__20230504.csv:9]: looks good, confirming rev k

File: calibration/PHSENF/CGINS-PHSENF-P0081__20230412.csv (hunk 16 above)
Comment (cdobs) [inline calibration/PHSENF/CGINS-PHSENF-P0081__20230412.csv:9]: looks good

File: calibration/PHSENF/CGINS-PHSENF-P0103__20230526.csv (hunk 17 above)
Comment (cdobs) [inline calibration/PHSENF/CGINS-PHSENF-P0103__20230526.csv:9]: looks good

File: calibration/PHSENF/CGINS-PHSENF-P0105__20230412.csv (hunk 18 above)
Comment (cdobs) [inline calibration/PHSENF/CGINS-PHSENF-P0105__20230412.csv:9]: looks good

File: calibration/PHSENF/CGINS-PHSENF-P0141__20230412.csv (hunk 19 above)
Comment (cdobs) [inline calibration/PHSENF/CGINS-PHSENF-P0141__20230412.csv:9]: looks good

File: calibration/PHSENF/CGINS-PHSENF-P0142__20230419.csv (hunk 20 above)
Comment (cdobs) [inline calibration/PHSENF/CGINS-PHSENF-P0142__20230419.csv:9]: looks good

File: calibration/PHSENF/CGINS-PHSENF-P0143__20230526.csv (hunk 21 above)
Comment (cdobs) [inline calibration/PHSENF/CGINS-PHSENF-P0143__20230526.csv:9]: looks good

**Review by cdobs — COMMENTED** [review]
All values look good. Just have questions about the format of DOSTA-D cal csvs. I left comments on each line. Once we confirm the current format is okay or clear it up I will approve

File: calibration/DOSTAD/CGINS-DOSTAD-00224__20230505.csv [hunk 25]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+224,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_224_Multipoint_Calibration_2023-05-05.pdf"
+224,CC_csv,"[2.82689E-03, 1.20089E-04, 2.27522E-06, 1.85755E+02, -2.27536E-01, -4.43958E+01, 3.63280E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00224__20230505.csv:3]: looks good!

File: calibration/DOSTAD/CGINS-DOSTAD-00379__20230505.csv [hunk 26]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+379,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_379_Multipoint_Calibration_2023-05-05.pdf"
+379,CC_csv,"[2.49522E-03, 1.08411E-04, 1.79483E-06, 2.04227E+02, -2.40200E-01, -4.66475E+01, 4.03495E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00379__20230505.csv:3]: looks good!

File: calibration/DOSTAD/CGINS-DOSTAD-00385__20230505.csv [hunk 27]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+385,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_385_Multipoint_Calibration_2023-05-05.pdf"
+385,CC_csv,"[2.56958E-03, 1.09730E-04, 2.06051E-06, 2.01644E+02, -2.95287E-01, -5.05581E+01, 4.03246E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00385__20230505.csv:3]: looks good!

File: calibration/DOSTAD/CGINS-DOSTAD-00386__20230505.csv [hunk 28]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+386,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_386_Multipoint_Calibration_2023-05-05.pdf"
+386,CC_csv,"[2.56178E-03, 1.09107E-04, 2.10725E-06, 5.75669E+01, -7.21496E-02, -1.42368E+01, 1.14979E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00386__20230505.csv:3]: looks good!

File: calibration/DOSTAD/CGINS-DOSTAD-00393__20230505.csv [hunk 29]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+393,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_393_Multipoint_Calibration_2023-05-05.pdf"
+393,CC_csv,"[2.60770E-03, 1.11383E-04, 2.11751E-06, 2.14793E+02, -3.11778E-01, -5.42762E+01, 4.27312E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00393__20230505.csv:3]: looks good!

File: calibration/DOSTAD/CGINS-DOSTAD-00413__20230505.csv [hunk 30]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+413,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_413_Multipoint_Calibration_2023-05-05.pdf"
+413,CC_csv,"[2.54000E-03, 1.08323E-04, 2.04878E-06, 1.79893E+02, -2.32320E-01, -4.25315E+01, 3.58009E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00413__20230505.csv:3]: looks good!

File: calibration/DOSTAD/CGINS-DOSTAD-00415__20230505.csv [hunk 31]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+415,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_415_Multipoint_Calibration_2023-05-05.pdf"
+415,CC_csv,"[2.60445E-03, 1.10694E-04, 2.09807E-06, 1.96260E+02, -2.68756E-01, -4.67264E+01, 3.90159E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00415__20230505.csv:3]: looks good!

**Review by cdobs — APPROVED** [review]

**Pull request merged**